corehq/apps/sms/migrations/0024_add_is_two_way.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from corehq.sql_db.operations import HqRunPython
+from django.db import migrations, models
+import datetime
+
+
+def set_pending_verification_numbers(apps, schema_editor):
+    PhoneNumber = apps.get_model('sms', 'PhoneNumber')
+    db_alias = schema_editor.connection.alias
+    PhoneNumber.objects.using(db_alias).filter(verified=False).update(is_two_way=False, pending_verification=True)
```

Comment:
yup, it does imply that right now, and that is done [here](https://github.com/dimagi/commcare-hq/pull/14506/files#diff-246f581da2e92dbc370d3d52c2277312R35)

basically the migration sets everything to two-way and then demotes the pending verification ones